src/InitialAnalyzer.cc

```diff
@@ -9,7 +9,7 @@
 #include <TFile.h>
 #include <iostream>
 #include <fstream>
-
+using namespace std;
```

Comment:
Please remove this change. It is bad practice to do this in general but you have it as a global.